packages/react-component-library/src/components/TopLevelNavigation/SidebarE/SidebarUserE.tsx

```diff
@@ -47,7 +47,9 @@ const SidebarAvatarWithItems: React.FC<SidebarAvatarWithItemsProps> = ({
         aria-label="Show user options"
         data-testid="user-button"
         icon={
-          <StyledUserAvatar initials={initials} data-testid="sidebar-avatar" />
+          <StyledUserAvatar data-testid="sidebar-avatar">
+            {initials}
+          </StyledUserAvatar>
         }
       />
     }
```

Comment:
I wasn't sure what to do with `SidebarAvatarWithItems` on line 78 as the pattern does not feel quite right but also it does not seem right to use `children` for the `initials`. Open to ideas

Reply:
Hmm maybe how that component is composed could be improved, but I'm not sure it's worth it as it's not exported as far as I can see (I think `initials` makes more sense than `children` there in any case).